### PR TITLE
Update/home add growth summit task

### PR DESCRIPTION
**Pull request by donlair**
This PR adds the client-side code for a new My Home card that promotes the upcoming Growth Summit. Additional context for this request is at pbvYe0-5F-p2. This won't go live until the backend code is deployed. This task card will only display to English-speaking users because the conference is English-only.

Here's a screenshot of the current design:
Screen Shot 2020-07-16 at 10 07 44 PM

The design and copy are still in progress and will be refined.

#### Testing instructions

* Checkout the branch (git checkout update/home-add-growth-summit-task)
* Patch your sandbox to D46543-code
* Update your hosts file to point API calls to your sandbox
* Confirm that your API calls are sandboxed (https://public-api.wordpress.com/?amisandboxed)
* Force the Growth Summit view to display on My Home (http://calypso.localhost:3000/home/SITESLUG?dev=true&view=VIEW_GROWTH_SUMMIT&flags=-home/layout-dev)

### CODE REVIEW

**Comment by gwwar** [comment]
@donlair you're always welcome to add PRs to the Serenity Team Board (so we don't lose track of pings). Don't forget to add a need review label to PRs though since this does trigger full desktop E2E tests.

File: client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ */
+import Task from 'my-sites/customer-home/cards/tasks/task';
+import { TASK_GROWTH_SUMMIT } from 'my-sites/customer-home/cards/constants';
+import growthSummitIllustration from 'assets/images/customer-home/illustration--growth-summit.svg';
+
+const GrowthSummit = () => {
+	return (
+		<Task
+			title="The WordPress.com Growth Summit"
+			description="Learn how to build and grow your site, from start to scale. Come join us for this exclusive two-day virtual event August 11-13."
```
Comment (gwwar) [inline client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx:17]: Please use `translate` as it's a good habit to stick to outside of example fragments. I understand we're targeting EN, though even that has US and GB variants.

File: client/assets/images/customer-home/illustration--growth-summit.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="244" height="164" viewBox="0 0 61 41" xmlns="http://www.w3.org/2000/svg"><path d="M2.003 41c-1.502 0-2.45-1.584-1.79-2.894l.072-.13 15.5-26c.748-1.255 2.527-1.3 3.349-.135l.087.135 4.84 8.119L35.288 1.468c.749-1.243 2.511-1.289 3.335-.138l.09.138 21.999 36.5c.777 1.29-.097 2.923-1.565 3.027L59 41H2.003z" fill="#fff"/><path fill="#009172" d="M37 2.5L59 39H35.33L25.222 22.043z"/><path fill="#5198D9" d="M17.503 13l5.388 9.037L12.667 39H2.003z"/><path fill="#EB6594" d="M24.052 23.985L33.003 39h-18z"/></svg>
```
Comment (gwwar) [inline client/assets/images/customer-home/illustration--growth-summit.svg:1]: Missing an endline

**Comment by gwwar** [comment]
Full E2E tests have unrelated breakages. (We do need to fix this more generally for better testing confidence.)

**Comment by donlair** [comment]
Thanks for the review @gwwar! I've added the calls for translations and swapped out the icon with one provided by @ballio2000. Here's a SS:

Screen Shot 2020-07-17 at 3 15 59 PM

@danhauk let me know if you'd like any design tweaks made.

**Comment by gwwar** [comment]
Thanks code changes look good! You should be all set to land this if folks didn't have any other feedback.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4113171

Thank you @donlair for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4113171).

**Pull request merged**